app/src/main/java/jp/developer/retia/frozenword/ui/watch/WatchScreen.kt

```diff
@@ -0,0 +1,166 @@
+package jp.developer.retia.frozenword.ui.watch
+
+import androidx.annotation.IntRange
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/verno3632/FrozenWord/blob/69693efc910066613382c0b4d00078525306e025/app/src/main/java/jp/developer/retia/frozenword/ui/watch/WatchScreen.kt#L3"></span>Imports must be ordered in lexicographic order without any empty lines in-between</td>
    </tr>
  </tbody>
</table>